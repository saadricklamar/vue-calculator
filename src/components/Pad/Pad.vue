<template>
  <section class="pad">
    <article class="buttons">
      <FirstRowButtons />
      <SecondRowButtons />
    </article>
  </section>
</template>

<script>
import FirstRowButtons from "../FirstRowButtons/FirstRowButtons";
import SecondRowButtons from "../SecondRowButtons/SecondRowButtons";

export default {
  name: "Pad",
  components: {
    FirstRowButtons,
    SecondRowButtons
  }
};
</script>

<style scoped>
.pad {
  display: flex;
  justify-content: center;
}

article {
  width: 425px;
  height: 400px;
  background-color: #556884;
  border-radius: 5px;
}
</style>