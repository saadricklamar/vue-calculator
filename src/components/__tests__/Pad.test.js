import { mount } from "@vue/test-utils";
import Pad from "../Pad/Pad";

describe("Pad", () => {
  const wrapper = mount(Pad);
  it("should match the snapshot", () => {
    expect(wrapper.isVueInstance()).toMatchSnapshot();
  });
});
