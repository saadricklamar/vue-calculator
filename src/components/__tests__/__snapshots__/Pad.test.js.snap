// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pad should match the snapshot 1`] = `true`;
